.github/workflows/e2e-stress-test-flake-fix.yml

```diff
@@ -20,6 +20,15 @@ on:
         type: string
         required: false
         default: "ee"
+      qa_db:
```

Comment:
Isolating this is absolutely the best, but we support mysql and we need to add it here as well.

Reply:
@nemanjaglumac Postgres and MySQL snapshots are [created together](https://github.com/metabase/metabase/blob/6045f2d22aea89e3689eed5251b731c7fa925f4b/e2e/snapshot-creators/qa-db.cy.snap.js#L23-L47). Maybe I should just change the option name to `postgres-mysql`? I don't want to modify the snapshot creation logic too much, since it's not entirely the scope of this PR.

Reply:
Got it.
Well, just change it to `sql`?

Reply:
brilliant!

Reply:
Fixed in https://github.com/metabase/metabase/pull/46065/commits/f4679130ae15ae99448dee4bdc74d0cd0cf5e50f